games-action/minetest/minetest-0.4.17.ebuild

```diff
@@ -0,0 +1,136 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit cmake-utils eutils gnome2-utils user
```

Comment:
no, `usex` is the part of `EAPI >=5`

Reply:
https://devmanual.gentoo.org/ebuild-writing/eapi/index.html

Reply:
Ah okay, I'll remove eutils as well then.